aggregator/src/aggregator.rs

```diff
@@ -212,7 +214,12 @@ pub struct Config {
     /// The key used to sign HPKE configurations.
     pub hpke_config_signing_key: Option<EcdsaKeyPair>,
 
+    /// Configuration for the taskprov extension.
     pub taskprov_config: TaskprovConfig,
+
+    /// If set, forbidden mutations of resources (e.g., re-using the same aggregation job ID but
+    /// with different reports in it) will be logged to files under this path when detected.
+    pub log_forbidden_mutations: Option<PathBuf>,
```

Comment:
Do we want to keep this option around long-term, or are we planning to back this change out once diagnosis is done? If the former, we may want to note that this option is not covered by our stability guarantees.

Reply:
Yeah, I think we should document that this isn't stable. If we do decide to keep it, we can always promote it to stable.